NEWS.md

```diff
@@ -1,3 +1,19 @@
+### v8.0.0 (2023-08-28)
+
+* **BREAKING**: Removed support for Node 14.
```

Comment:
we're missing a line item that says `Added support for Node 20.`

Reply:
Updated